openmdao/core/driver.py

```diff
@@ -288,6 +289,31 @@ def _setup_driver(self, problem):
         src_objs = prom2ivc_src_dict(self._objs)
         responses = prom2ivc_src_dict(self._responses)
 
+        # Only allow distributed design variables on drivers that support it.
+        if self.supports['distributed_design_vars'] is False:
+            dist_vars = []
+            for dv, meta in self._designvars.items():
+
+                # For Auto-ivcs, we need to check the distributed metadata on the target instead.
+                if meta['ivc_source'].startswith('_auto_ivc.'):
+                    abs_name = model._var_allprocs_prom2abs_list['input'][dv][0]
```

Comment:
Should we loop over these and set dist = True if any of them are distributed?  It seems unlikely but possible  that you could have an auto_ivc connected to multiple inputs where only some subset of them are distributed.

Reply:
I guess we allow a connection like that, so yeah: I need to loop over each one in the list and check.

Reply:
done